src/utils/time.ts

```diff
@@ -53,57 +86,23 @@ export const secondsToDuration = (totalSeconds: number): Duration => {
 export const formatSecondsAsDuration = (seconds: number): string =>
   formatDuration(secondsToDuration(seconds));
 
-const formatMeasurementInString = (value: number, measureUnit: string) => {
-  const baseFormatted = `${value} ${measureUnit}`;
-
-  if (value === 0) return "";
-  if (value === 1) return baseFormatted;
-  if (value > 1) return `${baseFormatted}s`;
-};
-
-// @debt quality test this
 export const getTimeBeforeParty = (startUtcSeconds?: number) => {
   if (startUtcSeconds === undefined) return "???";
 
-  const secondsBeforeParty =
-    startUtcSeconds - Date.now() / ONE_SECOND_IN_MILLISECONDS;
-
-  if (secondsBeforeParty < 0) {
-    return 0;
-  }
+  const eventStartDate = fromUnixTime(startUtcSeconds);
+  const now = Date.now();
 
-  const numberOfCompleteDaysBeforeParty = Math.floor(
-    secondsBeforeParty / ONE_DAY_IN_SECONDS
-  );
-
-  const numberOfCompleteHours = Math.floor(
-    (secondsBeforeParty % ONE_DAY_IN_SECONDS) / ONE_HOUR_IN_SECONDS
-  );
+  if (isBefore(eventStartDate, now)) return 0;
```

Comment:
[NB] This logic would probably be easier to read if it was `isAfter(now, eventStartDate)`

Reply:
[NB] For the locations that call this function, how are they using the return value? Can we return a string here without breaking things? `"0"`?

Reply:
refactored: https://github.com/sparkletown/sparkle/pull/1426/commits/64e2b9d2c1f4b2cf715ff86330b85947d012a966